### ISSUE TITLE
known_timers.h and known_boards.h not here on Github

### ISSUE THREAD

**Issue by GreyGnome** [PaulStoffregen/AltSoftSerial#12]
Hi,
I notice that in config there seem to be a couple of files that are in the Zip but not here on Github. Not sure if that's correct or not but I thought I'd point it out because I had a bit of trouble finding the definition of CAPTURE_INTERRUPT, for example.

Thanks.

**Comment by PaulStoffregen** [comment]
It's defined here.

https://github.com/PaulStoffregen/AltSoftSerial/blob/master/config/AltSoftSerial_Timers.h#L45

Are you sure there's something missing from github?  Can you be more specific?

**Comment by GreyGnome** [comment]
Ah, I see. I thought there was something missing because I did the
following:

   - Wondered how COMPARE_A_INTERRUPT was defined
   - Did a Google search (COMPARE_A_INTERRUPT altsoftserial), first result
   was with a site that pointed me to a site that had config/known_timers.h
   (->
   https://github.com/RedBearLab/BLE_HCI/blob/master/arduino/libraries/AltSoftSerial/config/known_timers.h
   )
   - Browsed to
   https://github.com/PaulStoffregen/AltSoftSerial/tree/master/config, did
   not see known_timers.h . Wondered where it came from.
   - Downloaded library zip file, it had the file in it under config.
   - Posted bug report.

I was confused because I found a file in the zip file under config that was
not on the Github site.

On Wed, May 20, 2015 at 11:51 AM, Paul Stoffregen <notifications@github.com>
wrote:

> It's defined here.
>
>
> https://github.com/PaulStoffregen/AltSoftSerial/blob/master/config/AltSoftSerial_Timers.h#L45
>
> Are you sure there's something missing from github? Can you be more
> specific?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PaulStoffregen/AltSoftSerial/issues/12#issuecomment-103956969>
> .
>



-- 
-Mike Schwager
